com.microsoft.mrtk.graphicstools.unity/Runtime/MeshOutline/MeshOutline.cs

```diff
@@ -13,15 +13,15 @@ namespace Microsoft.MixedReality.GraphicsTools
     /// This behavior is designed to be used in conjunction with the Graphics Tools/Standard shader. Limitations of this effect include it not working well 
     /// on objects which are not watertight (or required to be two sided) and depth sorting issues can occur on overlapping objects.
     /// </summary>
-    [RequireComponent(typeof(MeshRenderer))]
+    [RequireComponent(typeof(Renderer))]
     [AddComponentMenu("Scripts/GraphicsTools/MeshOutline")]
     public class MeshOutline : BaseMeshOutline
     {
         private const string vertexExtrusionKeyword = "_VERTEX_EXTRUSION";
         private const string vertexExtrusionSmoothNormalsKeyword = "_VERTEX_EXTRUSION_SMOOTH_NORMALS";
         private const string vertexExtrusionValueName = "_VertexExtrusionValue";
 
-        private MeshRenderer meshRenderer = null;
+        private Renderer meshRenderer = null;

```

Comment:
nit: I wonder if we should change the variable name from `meshRenderer` to something else now that it is more generic?